Packs/Okta/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -824,11 +833,21 @@ def set_password_command(client, args):
 
 def expire_password_command(client, args):
     user_id = client.get_user_id(args.get('username'))
+    hide_password = args.get('hide_password')
 
     if not (args.get('username') or user_id):
         raise Exception("You must supply either 'Username' or 'userId")
 
-    raw_response = client.expire_password(user_id, args)
+    if args.get('revoke_session') == 'true':
+        raw_response = client.revoke_session(user_id, args)
```

Comment:
I think you do not need to add a new endpoint but add the {revokeSessions: args.get('revoke_session', 'false')} as an additional param in expire_password function- please test it that way and let me know.

Reply:
Following Okta's documentation it looks like the Revoke Sessions Parameter is only available under the expirePasswordWithTempPassword endpoint. 

https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserCred/#tag/UserCred/operation/expirePasswordWithTempPassword

The expirepassword endpoint has no parameters listed outside of the path parameter.

https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserCred/#tag/UserCred/operation/expirePasswordWithTempPassword

I added the revoke_session method to stay true to their documentation. Let me know if you'd still like me to proceed with your recommended modification.

Reply:
Thank you for the links!
I think some of the the parameters may not be documented in their site.
In the current code in the expire_password endpoint we may send the tempPassword parameter which is also not documented, thats why i wanted you please to try if we can also add the revokeSessions parameter (just for code reuse, if not- will leave it that why).
Thank you so so much!

Reply:
You are correct, revoke sessions works with the expirePassword endpoint. I just pushed the change and included a more logic in the conditional statement for the hide password argument. We only need to hide the password if temporary_password is True.

Reply:
Great work!
I cannot see the the addition to the expire_password function. Am i missing something?
the addition should look something like this:
if revoke_session:= args.get('revoke_session'):
params['revokeSession']='true'
what do you think?

Reply:
@tcarmeli1 Thank you for catching that. I presumed the args would get passed to the method, but forgot about setting the params. Looking at the logs in my Okta trial, I don't see the users sessions getting cleared like I did on October 21st before I removed the revoke_session method. 

Seeing this it appears we need to use expirePasswordWithTempPassword endpoint to revoke the user sessions. I'll add the method back to the integration.